### ISSUE TITLE
Simplfy the licence component of gofed spec generation

### ISSUE THREAD

**Issue by nullr0ute** [gofed/gofed#34]
The use of %license in the gofed generated spec files is a lot more complex than it needs to be in an attempt to deal with EL6 distros. According to the EPEL packaging guidelines [1] all that needs to be done is to define %license to be the same as %doc:

[1] http://fedoraproject.org/wiki/EPEL:Packaging#The_.25license_tag

So all that we need to do is actually the following:

``` vim
-%define copying() \
-%if 0%{?fedora} >= 21 || 0%{?rhel} >= 7 \
-%license %{*} \
-%else \
-%doc %{*} \
-%endif
+%{!?_licensedir:%global license %%doc}
```

And then in the %files section all we need is (obviously adjusting for the appropriate actual license):
-%copying LICENSE.txt
+%license LICENSE.txt

The above change makes it quite a bit simpler and makes it more standards complaint to the packaging guidelines.

**Comment by ingvagabund** [comment]
The solution is not working, I am getting:
```vim
error: File must begin with "/": MIT
error: File must begin with "/": LICENSE.md
```
for
```vim
%license LICENSE.md
```
I have a feeling it conclicts with License tag.

**Comment by ingvagabund** [comment]
When I add %debug and %trace to spec file, I am getting:
```vim
  1>   %files^
  0< %files
  1>   %license^LICENSE.md
  1<   MIT
  0< MIT LICENSE.md
```

%license is replaced by license tag, not by license macro :(.

**Comment by ingvagabund** [comment]
Why did I put it before license tag is defined? :) That was the reason why it did not work. It must be defined after license tag is. So the best position is before the first %files section.

**Comment by ingvagabund** [comment]
Confirming scratch build is working. Fixed via ecf6eb81990cb41f97598d8c4a224182bded9815.

Thanks Peter.
